backend/go.mod

```
@@ -5,10 +5,9 @@ go 1.25.0
 require (
 	entgo.io/ent v0.14.4
 	github.com/GoYoko/web v1.4.0
```

更新依赖版本是个好主意，但需要确保这些更新不会引入不兼容的变更或安全问题。建议检查 ModelKit 从 v2.0.6 到 v2.3.1 的变更日志。

```suggestion
// 检查 ModelKit 的变更日志以确保兼容性
// 可以在 https://github.com/chaitin/ModelKit/releases 查看详细信息